### PR TITLE
Improve the Readme of product_sequence: add 'Try me on Runbot'

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

File: product_sequence/README.rst [hunk 1]
```diff
@@ -13,6 +17,20 @@ they already have a unique reference (or no reference) set.  Products with a def
...
 Otherwise the setting of the unique constraint will fail and the module will fail to install.
 
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/8.0
+
+.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: Remove these lines

File: product_sequence/README.rst [hunk 2]
```diff
@@ -13,6 +17,20 @@ they already have a unique reference (or no reference) set.  Products with a def
...
+.. branch is "8.0" for example
+
+For further information, please visit:
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: Ans also these useless ones

File: product_sequence/README.rst [hunk 3]
```diff
@@ -1,3 +1,7 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```
Comment (pedrobaeza) [inline product_sequence/README.rst]: Add target link

**Comment by lmignon** [comment]
@pedrobaeza done

**Comment by pedrobaeza** [comment]
:+1:

**Comment by eLBati** [comment]
:+1:

**Pull request merged**